packages/web/src/scss/components/Modal/README.md

```diff
@@ -14,14 +14,31 @@ Modal establishes the top layer with a backdrop. Under the hood it uses the [`<d
 provides several accessibility advantages.
 
 ```html
-<dialog id="modal-example" class="Modal" aria-labelledby="modal-example-title">
+<dialog id="modal-example" class="Modal Modal--center" aria-labelledby="modal-example-title">
```

Comment:
is this required if it is the default?

Reply:
Because if so, it looks like a BC to me.

Reply:
It's a default value that will be supplied by default props in Twig and React. It's equivalent to `.ModalFooter--right` or`.Button--medium`.

It's not a BC because:

```scss
.Modal--center,
.Modal:not(.Modal--top, .Modal--bottom) {
  // …
}
```

Reply:
ok